src/release/main.d

```diff
@@ -335,6 +332,28 @@ ActionList prepareMinorRelease ( ref HTTPConnection con, ref Repository repo,
 
 /*******************************************************************************
 
+    Prepares the actions to clar the release notes
+
+    Returns:
+        actions required to clear the release notes
+
+*******************************************************************************/
+
+ActionList clearReleaseNotes ( )
```

Comment:
You mean if you want to check it via submodule? It is still not that hard, pretty much `cd submodules/stuff; git show HEAD`. I wonder though what approach is more common - that or reading notes in web release page/mail.

Problem with keeping notes is that we have to either manually separate them after merging into next branch or merge everything twice - first actual released tag and "clean notes" branch commit right after.